lib/bson/error/invalid_decimal128_range.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module BSON
+  class Error
+
+    # Raised when the exponent is outside the valid range.
+    class InvalidDecimal128Range < Error
+
+      # The custom error message for this error.
+      #
+      # @deprecated
+      MESSAGE = 'Value out of range for Decimal128 representation.'
```

Comment:
If this and other deprecated constants aren't used, maybe get rid of them completely?

Reply:
this looks like its used in `def message`

Reply:
I'd like these constants removed from the codebase and exception messages provided as literal strings everywhere, if you agree I'll make a ticket for this.